rel/sys.config

```diff
@@ -1,4 +1,10 @@
 [
+ {sync, [
+        {src_dirs, {replace, [
+				{"../../applications", []},
```

Comment:
Good then :) 
The empty lists at `{"../../applications", []}` and the core one. Reading the `sync` doc I understood that's where compile options should be added. Maybe `sync` gets the compilation options from the BEAM chunks already? (That would be great)